weasyl/controllers/decorators.py

```diff
@@ -36,7 +36,7 @@ def inner(request):
         if request.userid not in staff.MODS:
             return Response(define.errorpage(request.userid, errorcode.permission))
         return view_callable(request)
-    return inner
+    return login_required(inner)
```

Comment:
Both of these decorators now need to have docstrings that mention that they imply login required. It's too easy for someone glancing through and recognizing a decorator to gloss over the actual return line.

(also it'll show up in any docs we generate and in `help()` that way)